arch/sim/include/setjmp.h

```diff
@@ -97,12 +101,42 @@
 #  define JB_FP               JB_EBP
 
 #elif defined(CONFIG_HOST_ARM)
+
 #  define XCPTCONTEXT_REGS    16
 #  define XCPTCONTEXT_SIZE    (4 * XCPTCONTEXT_REGS)
 
 #  define JB_FP               7
 #  define JB_SP               8
 #  define JB_PC               9
+
+#elif defined(CONFIG_HOST_ARM64)
+
+#  define XCPTCONTEXT_REGS    32
+#  define XCPTCONTEXT_SIZE    (8 * XCPTCONTEXT_REGS)
+
+#  ifdef __ASSEMBLY__
+
+#    define	JB_X19_X20	      #0x00
+#    define	JB_X21_X22	      #0x10
+#    define	JB_X23_X24	      #0x20
+#    define	JB_X25_X26	      #0x30
+#    define	JB_X27_X28	      #0x40
+#    define	JB_X29_XLR  	    #0x50
+#    define	JB_XFP_XSP	      #0x60
+
+#    define	JB_D08_D09	      #0x70
+#    define	JB_D10_D11	      #0x80
+#    define	JB_D12_D13	      #0x90
+#    define	JB_D14_D15	      #0xA0
+
```

Comment:
move the define to assemble file?

Reply:
I get your point, but as these values needs to be aligned with JB_PC, JB_FP and JB_SP they are kept in one file. This is the same as for X86 definitions.